server.js

```diff
@@ -1,23 +1,99 @@
 const express = require("express");
 const cors = require("cors");
+const bodyParser = require("body-parser");
```

Comment:
Why are you importing this library? It is not normally included in modern express apps: https://medium.com/@mmajdanski/express-body-parser-and-why-may-not-need-it-335803cd048c

Reply:
Hi James,
Thank you so much for link.
I haven't known it before.
'express 4.16. '